src/app/pages/privacy-policy/privacy-policy.component.html

```diff
@@ -0,0 +1,45 @@
+<app-hill-background cover></app-hill-background>
+<div class="home-container">
+  <mat-card class="column">
+    <mat-card-title>
+      <h1 style="font-size:60px;" class="title">Privacy Policy</h1>
```

Comment:
This title is way too big and looks bad on mobile. Please use the proper [Angular Material typography classes](https://material.angular.io/guide/typography) for headers.